assets/kube-apiserver/kube-apiserver-deployment.yaml

```diff
@@ -111,6 +111,6 @@ spec:
           name: kube-apiserver-vpnclient-config
         name: vpnconfig
       - secret:
-          secretName: openvpn-client
+          secretName: kube-apiserver-vpnclient-secret
```

Comment:
This will break our automation as well.

Reply:
so you're rendering with the --include-vpn flag?